dist/obs-server.spec

```diff
@@ -831,6 +831,9 @@ usermod -a -G docker obsservicerun
 
 /srv/www/obs/api/config/boot.rb
 /srv/www/obs/api/config/routes.rb
+/srv/www/obs/api/config/routes/api_routes.rb
```

Comment:
You should add the whole directory, not just the files in it.

Reply:
nah, it must not overwrite %config definitions for other files inside

Reply:
I don't think config/routes should be %confg

Reply:
done @hennevogel , looks package is building successfully.